values.yaml

```diff
@@ -247,17 +247,27 @@ ingress:
 
 # -- https://github.com/bitnami/charts/tree/master/bitnami/elasticsearch#parameters
 elasticsearch:
-  # `false` will disable full-text search
+  # Elasticsearch is powering full-text search. It is optional.
+
+  # `false` will not install Elasticsearch as part of this chart
   #
   # if you enable ES after the initial install, you will need to manually run
   # RAILS_ENV=production bundle exec rake chewy:sync
   # (https://docs.joinmastodon.org/admin/optional/elasticsearch/)
-  # @ignored
   enabled: true
   # @ignored
   image:
     tag: 7
 
+  # If you are using an external ES cluster, use `enabled: false` and set the hostname and port
+  # hostname:
```

Comment:
We should probably add a comment here about requiring `https://` if you're using TLS for elasticsearch

Reply:
Ah that's a good call

Reply:
Maybe even introduce a value parameter for like `elasticsearch.use-tls` and if it is true prepend it.

That way we take away the domain knowledge required for setting the host name.